packages/color-block/src/components/cards/CardsItem.tsx

```diff
@@ -74,6 +74,7 @@ export const CardsItem = ({ color, colorSpaces, isEditing, onBlur, onUpdate, onD
                     }
                     triggerElement={
                         <div className="tw-overflow-hidden tw-rounded-t tw-bg-[url('https://cdn.frontify.com/img/transparent.png')] tw-bg-[length:10px_10px]">
+                            {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events,jsx-a11y/no-static-element-interactions */}
```

Comment:
add role="button" and a tabIndex={0} to overcome the second rule. For the first rule, maybe it's worth to add an onkeydown listener to Enter here as well.

Reply:
Won't tackle as its an unreleased block and needs some work anyway before go live